RNA-seq/03-gastric_cancer_exploratory.Rmd

```diff
@@ -166,30 +128,41 @@ ggplot2::ggsave(pca_plot_file, plot = ggplot2::last_plot())
 We don't have batch information (i.e., when the samples were run) for this 
 particular experiment, but let's imagine that `SRR585574` and `SRR585576` were
 run separately from all other samples.
-We'll put this in new "toy" column in `sample_meta_df`.
+We'll add this as a new "toy" column in the sample data (`colData`) for `gene_summarized`.
+
+```{r extract-sample, live = TRUE}
+# Extract colData
+sample_info <- colData(gene_summarized)
```

Comment:
For clarity with later, maybe we should pull the sample info from `vst_data`, as that is where we will be putting this back. They are identical, I think, but this might cause less confusion.
```suggestion
sample_info <- colData(vst_data)
```

Reply:
Ah. Yes. That would be less confusing.